easybuild/easyconfigs/l/libevent/libevent-2.1.8.eb

```diff
@@ -0,0 +1,19 @@
+easyblock = 'ConfigureMake'
+
+name = 'libevent'
+version = '2.1.8'
+
+homepage = 'http://libevent.org/'
+description = """The libevent API provides a mechanism to execute a callback function when a specific
+ event occurs on a file descriptor or after a timeout has been reached.
+ Furthermore, libevent also support callbacks due to signals or regular timeouts."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = [
+    'https://github.com/downloads/%(name)s/%(name)s/',
+    'https://sourceforge.net/projects/levent/files/%(name)s/%(name)s-%(version_major_minor)s/',
+]
+sources = ['%(name)s-%(version)s-stable.tar.gz']
+
```

Comment:
@foxik0070 Are you up for including a custom `sanity_check_paths` here, to have a better check for whether what `libevent` is supposed to provide is really there?

We could then also include this in the other existing `libevent` easyconfigs (probably best in a separate PR).